packages/template-retail-react-app/app/constants.js

```diff
@@ -60,3 +60,5 @@ export const urlPartPositions = {
 }
 
 export const DEFAULT_SITE_TITLE = 'Retail React App'
+
+export const CACHE_AGE = 60 * 15 // 15 min
```

Comment:
```suggestion
export const MAX_CACHE_AGE = 60 * 15 // 15 min
```
Might make it a bit more clear what "cache age" actually is.